client-app/app/controllers/index.js

```diff
@@ -13,6 +13,8 @@ import { tracked } from "@glimmer/tracking";
 @classic
 export default class IndexController extends Controller {
   @tracked loading = false;
+  @tracked buildingGroupingRegex = false;
+  @tracked rowMessagesForGroupingRegex = [];
```

Comment:
Don't you need TrackedArray here? (since you're pushing to this array, not just replacing it)

Reply:
I don't believe so, it is working as expected locally.